src/main/resources/frontend/src/types/types.ts

```diff
@@ -40,7 +40,17 @@ export type RubricItem = {
     results: RubricItemResults,
 }
 
+//unfortunately with the way gson serializes maps and the way javascript parses that json each needs to be its own item
+export type RubricItems = {
```

Comment:
Consider using the builtin Typescript `Record` type to represent this more cleanly. If we already had a `type` that represents the different kinds of Phases, using that would be ideal, but I'm not sure if/where it exists.

```ts
export type RubricItems = Record<"PASSOFF_TESTS"|"UNIT_TESTS"|"QUALITY"|"GIT_COMMITS", RubricItem>;
export type RubricItems = Record<PhaseNames, RubricItem>; // Preferred, if available
```

Reply:
Thank you. That provides the exact same functionality I attempted and failed to reproduce with a map